catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -171,6 +182,12 @@
 
     @Override
     protected Object executeWithSubject() throws Exception {
+        if (batchSize * multithreaded > MAX_QUEUE_SIZE) {
```

Comment:
Made this change to protect from too large a queue of metacards and to avoid the complexity of managing tiered Phasers.